resources/hpxml-measures/HPXMLtoOpenStudio/resources/hpxml_defaults.rb

```diff
@@ -142,7 +142,7 @@ def self.apply_header(hpxml_header, epw_file)
     end
 
     if hpxml_header.temperature_capacitance_multiplier.nil?
-      hpxml_header.temperature_capacitance_multiplier = 1.0
+      hpxml_header.temperature_capacitance_multiplier = 7.0
```

Comment:
Recall I made a commit where we set this to 7.0 using the ResStockArguments measure; I don't think we need this change to `hpxml_defaults.rb`.